sdk/python/feast/infra/contrib/spark_kafka_processor.py

```diff
@@ -68,13 +67,10 @@ def __init__(
         # data_source type has been checked to be an instance of KafkaSource.
         self.data_source: KafkaSource = self.data_source  # type: ignore
 
-    def ingest_stream_feature_view(
-        self, to: PushMode = PushMode.ONLINE
-    ) -> StreamingQuery:
+    def ingest_stream_feature_view(self, to: PushMode = PushMode.ONLINE) -> None:
```

Comment:
Why should we modify the function behaviour 🤔 ?

Reply:
The base class does not return anything, I'm just following its signature.

Reply:
I see, clear

Reply:
I reverted this change because I don't know enough about the codebase. Looking at the pyspark.StreamingQuery class, some users might be actively using the return type, for example, to stop the query.

Reply:
I changed the signature to return Optional[Any] instead to fix the linting issue.